velox/functions/sparksql/Register.cpp

```diff
@@ -124,7 +124,7 @@ void registerFunctions(const std::string& prefix) {
   // Register size functions
   registerSize(prefix + "size");
 
-  registerFunction<JsonExtractScalarFunction, Varchar, Varchar, Varchar>(
```

Comment:
Are these any remaining usage of JsonExtractScalarFunction? If not, let's remove it.

Reply:
What is JsonExprBenchmark.cpp for? If it is just for benchmarking this function, then since function is no longer used, the benchmark is no longer needed and can be removed.

Reply:
@mbasmanova, I found [JsonExprBenchmark.cpp](https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/benchmarks/JsonExprBenchmark.cpp#L34) is used to benchmark a set of functions. Maybe, we can keep it in the source code?

Reply:
@PHILO-HE We can keep the benchmark, but there is no point in benchmarking unused JsonExtractScalarFunction, hence, let's remove both the function and benchmark code.

Reply:
Alternatively, we can move JsonExtractScalarFunction into the benchmark, assuming it is used to compare folly-based and simd-based implementations.

Reply:
Just removed `JsonExtractScalarFunction` and the related usage in benchmark. Thanks!